elements/bulbs-video/elements/video-carousel.js

```diff
@@ -68,8 +71,9 @@ class BulbsVideoCarousel extends BulbsHTMLElement {
 
   firstPlay () {
     let items = this.querySelectorAll('bulbs-carousel-item');
+    let nowPlayingItems = this.querySelectorAll('[now-playing]');
 
-    if (items.length > 0) {
+    if (nowPlayingItems.length > 0 && items.length > 0) {
```

Comment:
but the `selectItem` call below is currently forcing `items[0]` to play, regardless of which one is selected. the highlight styles don't seem to be related to `firstPlay`, as this is only called once (and then the eventlistener is removed), so I'm not really sure what the intended functionality of this is